rolecommands/bot.go

```diff
@@ -51,15 +51,16 @@ func (p *Plugin) AddCommands() {
 		RequireDiscordPerms: []int64{discordgo.PermissionManageServer},
 		RequiredArgs:        1,
 		Arguments: []*dcmd.ArgDef{
-			{Name: "Group", Type: dcmd.String},
+			{Name: "Group", Help: "The role command group", Type: dcmd.String, Autocomplete: true},
 		},
 		ArgSwitches: []*dcmd.ArgDef{
 			{Name: "m", Help: "Message ID", Type: dcmd.BigInt},
 			{Name: "nodm", Help: "Disable DM"},
 			{Name: "rr", Help: "Remove role on reaction removed"},
 			{Name: "skip", Help: "Number of roles to skip", Default: 0, Type: dcmd.Int},
 		},
-		RunFunc: cmdFuncRoleMenuCreate,
+		RunFunc:          cmdFuncRoleMenuCreate,
+		AutocompleteFunc: acFuncRoleMenuCreate,
```

Comment:
I think this should be part of ApplicationCommandOption instead of ApplicationCommand, because if there are more than one Autocomplete arguments it becomes a pain to extend with this.